<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control</class>
 <widget class="QMainWindow" name="Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skin Detector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="train_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bounding Cube size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Skin Detector</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_model_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Models</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_model_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Models</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_record_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>view record</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Mixture Models</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Image set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_camera">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize camera</string>
    </property>
   </widget>
   <widget class="QWidget" name="with_cam" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QPushButton" name="stop_camera">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>440</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="record_button">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_record_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>280</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>431</width>
       <height>401</height>
      </rect>
     </property>
     <widget class="QSlider" name="z_max_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="y_min_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="x_min_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="x_max_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="y_max_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="z_min_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>351</width>
        <height>29</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min X (pixels)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max X (pixels)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Y (pixels)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Y (pixels)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min Z (cms)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Z (cms)</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_min_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_max_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_min_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_max_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_min_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_max_label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>350</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>with_cam</zorder>
   <zorder>train_button</zorder>
   <zorder>test_button</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>save_model_button</zorder>
   <zorder>load_model_button</zorder>
   <zorder>view_record_button</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>start_camera</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
